packages/bpk-component-dialog/src/BpkDialog.js

```diff
@@ -46,6 +46,7 @@ const BpkDialog = (props: Props) => {
       closeOnScrimClick={dismissible}
       closeOnEscPressed={dismissible}
       fullScreenOnMobile={false}
+      isIphone={false}
```

Comment:
Do we not have snapshot tests? If not can we add some please? 😇

Reply:
We do have snapshots, I think it just hasn't caused the markup to change for whatever reason.